java/Makefile

```diff
@@ -1,4 +1,5 @@
 init:
+	./setup-java.sh
```

Comment:
I could not figure out a way to get `make` to be happy about using `sdk` without complaining about permissions. To work around that I just created a little `setup-java.sh` script to install Java 21 and then set it as the default within the codespace.